src/components/encounter/encounter-form-manager.ts

```diff
@@ -98,6 +106,58 @@ export class EncounterFormManager {
       return savePatientIdentifier(patientIdentifier, patient.id);
     });
   }
+
+  static prepareProgramEnrollment(
+    fields: FormField[],
+    encounterLocation: string,
+    patient: fhir.Patient,
+    encounterContext: EncounterContext,
+    patientPrograms: Array<PatientProgram>,
+  ) {
+    const programUuid = fields.find((field) => field.type === 'programState')?.questionOptions.programUuid;
```

Comment:
For now the assumption is that a form is affiliated to a program because we are making one request to update/enroll a patient along with the state, which is part of that program